### ISSUE TITLE
Nanoscopic examples is not working properly

### ISSUE THREAD

**Issue by nbeloglazov** [quil/quil-site#30]
After 3.0.0 the example stopped working propery: key presses don't have any effect
http://quil.info/sketches/show/example_nanoscopic

**Comment by nbeloglazov** [comment]
@anthonygalea do you know of any issues related to key-pressed handler in p5js?

**Comment by anthonygalea** [comment]
Not aware of any issues but what I just noticed is that when the canvas has focus it works. Haven't had time to check this issue but I have a vague recollection of seeing a commit related to global key events while working on the migration.

**Comment by anthonygalea** [comment]
Ah yes, this is [the commit](https://github.com/quil/quil/commit/189b2a8d0f8560b9ac246a23204ac01e48057878). Also, in this [p5 example](https://p5js.org/examples/input-keyboard.html) it explicitly states "Click on the image to give it focus". Can it be that global is no longer an option as before?

**Comment by nbeloglazov** [comment]
ohhh. Actually I have different problem. I tried now and on macos chrome WASD keys work to steer the  ship but left/right/up/down arrow keys don't work.

**Comment by anthonygalea** [comment]
Yes, for me it's the same, I had only tried WASD keys :) I think it boils down to `key-coded?` always returning `false` when called from [key-as-keyword](https://github.com/quil/quil/blob/master/src/cljc/quil/core.cljc#L4949). See [this sketch]( http://quil.info/sketches/show/f54a2165c2d714ad02e18b2d5eb566e2f5c49038f660c162b5c58eadff8d210d). Not really understanding [this logic](https://github.com/quil/quil/blob/master/src/cljc/quil/core.cljc#L4935).

While testing also found a bug which was fixed with `0.8.0` release of p5js. Under the manual tests i.e. http://localhost:3000/manual, the `mouse-and-key-pressed-variable` test, always displays `key-pressed` as `false`. Submitted https://github.com/cljsjs/packages/pull/1861. Once that's merged we can consider upgrade within quil.
